index.ts

```diff
@@ -276,6 +279,12 @@ function getMethod<
   TPublicMethod extends PublicMethod<TMethod>
 >(type: TType, options: Options = {}): TPublicMethod {
   const publicMethod = (...args: Parameters<TMethod>) => {
+    const handler = handlers.get(type);
+    if (handler) {
+      console.log("Messenger:", type, "is being handled locally");
+      return handler.apply({ trace: [] }, args);
+    }
```

Comment:
This is all it takes to handle messages locally (background only for now)

As you can see the `trace` array will be empty. The `whoAmI()` function for example will return `undefined` (if coded correctly) or throw an error.

Reply:
This was slightly updated to handle every background call, even if the handler is missing.